server/app/services/question/QuestionAnswerer.java

```diff
@@ -88,11 +88,13 @@ public static void answerNameQuestion(
       Path contextualizedPath,
       String first,
       String middle,
-      String last) {
+      String last,
+      String suffix) {
```

Comment:
**nit** I think setting the type of the `suffix` parameter to the `SuffixEnum` so it can only accept correct values will reduce the chance for bugs. It could be an `optional` if we need to not always set a suffix. Although since this is test code I could be convinced otherwise, but I think in prod code we should use the enum type everywhere possible.

Similarly, anywhere we take a string that is only ever going to be a Suffix, we should try to use the enum. Here, for example:
https://github.com/civiform/civiform/blob/bad18e641f95f7cbd90a699ba7a259af2ab4be93/server/app/models/ApplicantModel.java#L54

Reply:
I think since it's test code and this method hasn't being used outside of testing, so maybe using string is fine. But will def apply the enum type everywhere possible in prod code.